.github/workflows/update-latest-report.yml

```diff
@@ -0,0 +1,72 @@
+name: "Update Latest Report"
+
+on:
+  workflow_dispatch:
+    inputs:
+      which-gap:
+        description: 'Either a GAP branch name or a GAP version'
+        required: true
+        type: string
+        default: master # or 4.11.1 or ...
+  workflow_call:
+    inputs:
+      which-gap:
+        description: 'Either a GAP branch name or a GAP version'
+        required: true
+        type: string
+        default: master # or 4.11.1 or ...
+
+jobs:
+  test-all:
+    name: "Test packages and generate report"
+    uses: ./.github/workflows/test-all.yml
+    with:
+      which-gap: ${{ github.event.inputs.which-gap || inputs.which-gap }}
```

Comment:
@fingolfin, this line throws an error. I don't know why. Can you help me with this?
```
The workflow is not valid. .github/workflows/update-latest-report.yml (Line: 24, Col: 18):
Unrecognized named-value: 'inputs'.
Located at position 34 within expression: github.event.inputs.which-gap || inputs.which-gap
```

Reply:
So I used a `workflow_dispatch` to start the workflow and if I understand correctly, it tells me that `inputs.which-gap` does not exist. But I do not understand why the same syntax works in test-all.yml for example here:
``` yml
- name: "Upload GAP with packages as artifact"
        uses: actions/upload-artifact@v2
        with:
          name: gap-${{ github.event.inputs.which-gap || inputs.which-gap }}
          path: /home/runner/gap.tar.zst
```

Reply:
I see that the difference between these two is, that in the first (failing) example I call a workflow directly as in ["Reusing Workflows"](https://docs.github.com/en/actions/using-workflows/reusing-workflows#example-caller-workflow) and in the second (succeeding) example I call an action from within a step. Still, the error is weird.

Reply:
Oh, i just see in the [documentation](https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations) the following:

`Reusable workflows can't call other reusable workflows.`

Reply:
Ok, I fixed the issue by adjusting the workflow logic. But the error was really pointing me into a wrong direction. I don't really get why GitHub is making such a restriction in the first place, but whatever. Now it is working.